docs/other/troubleshooting-gratia.md

```diff
@@ -7,7 +7,11 @@ This document will help you troubleshoot problems with the Gratia Accounting, pa
 Gratia/GRACC: The Big Picture
 -----------------------
 
-Gratia is software used in OSG to gather accounting information. The information is collected from individual resources at a site, such as a Compute Entrypoint or a a submission host. The program that collects the data is called a "Gratia probe". The information is transferred to a GRACC server. Most sites will choose to send the accounting data to the central OSG Gratia server, but you can also use a Gratia server at your site (which can send forward the data to the central OSG Gratia server). Here is a diagram:
+Gratia is software used in OSG to gather accounting information.
```

Comment:
```suggestion
Gratia is software used in OSG to account for usage of computational resources.
```

Reply:
I actually prefer the original phrasing.  (It's definitely not something i would change in terms of correctness.)

If you want to say something about "computational resources" specifically, maybe we could say "to gather accounting information for usage of computational resources."

Also I haven't decided how i feel about describing "storage probes" as collecting usage info about "computational resources."

Reply:
> "to gather accounting information for usage of computational resources."

Fine by me.

> Also I haven't decided how i feel about describing "storage probes" as collecting usage info about "computational resources."

I don't think we have to worry about the storage probe case.

Reply:
ok